fluent-plugin-prometheus.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "fluent-plugin-prometheus"
-  spec.version       = "1.8.4"
+  spec.version       = "1.9.0"
```

Comment:
Don't include version update in feature request patch.

Reply:
Just to be sure, you want this removed and you will yourself do a commit for version update and changelog? Or should I do another dedicated commit?